Dockerfile

```diff
@@ -15,25 +15,35 @@ RUN apt-get update \
         build-essential \
         curl \
         default-jdk \
+        default-jre \
```

Comment:
isn't that part of jdk?

Reply:
they've got it listed separately

Reply:
but yeah, I think you're right, it's included in the jdk